tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -689,9 +689,53 @@ Validate basic geographic chart preview
 
     user checks map chart indicator tile contains    id:chartBuilderPreview    Admissions in 2005    3,612
 
+Change geographic chart data groupings
```

Comment:
Wouldn't it be better to have this before `Validate basic geographic chart preview`?

Feels like we'd want to add some extra checks for the legend and data grouping changes to ensure they look correct in the preview.

Reply:
I've added tests for checking the data groupings are shown correctly and update now, had to change some of the locations used in the chart because of lack of data for the ones it was using before.